src/cmdict/pdf_tools.py

```diff
@@ -47,6 +47,9 @@ def extract_words(file_path, color):
         for word in _fix_hyphen_broken(word_list):
             res.add(remove_punctuation(word))
 
+    c = PREVIEW_COLORS[color.lower()]
+    str_color = f"\033[38;2;{c[0]};{c[1]};{c[2]}m"
+    print(str_color + str(res) + "\x1b[0m")
```

Comment:
Basically you should not print anything from this file. You could echo just the color name instead of the content, in the `run_scripts`.